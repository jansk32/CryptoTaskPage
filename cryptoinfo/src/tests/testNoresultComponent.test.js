import React from 'react';
import ErrorComponent from '../noResultsComponent'
import renderer from 'react-test-renderer';


test("Testing for CryptoList No Result", () => {
    const component = renderer.create(
        <ErrorComponent error="noResult" />
    )

    expect(component.toJSON()).toMatchSnapshot();
})


test("Testing for No Connection", () => {
    const component = renderer.create(
        <ErrorComponent  />
    )

    let tree = component.toJSON();

    expect(tree).toMatchSnapshot();
})