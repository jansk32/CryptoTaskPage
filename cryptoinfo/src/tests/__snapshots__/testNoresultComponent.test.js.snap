// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing for CryptoList No Result 1`] = `
<tr>
  <td
    colSpan="7"
  >
    <center>
      <h1>
        NO RESULTS FOUND!
      </h1>
      <p>
        Try another search word...
      </p>
    </center>
  </td>
</tr>
`;

exports[`Testing for No Connection 1`] = `
<div>
  <center>
    <h1>
      There seems to be a problem
    </h1>
    <h1 />
    <h1>
      Please check your connection and try again...
    </h1>
    <img
      alt="cool coin"
      src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTTiSHJVnvVxKLiLbOen7dubuWq8H7Za8jjyw&usqp=CAU"
    />
    <p>
      In the meantime, here's a cool bitcoin for your trouble
    </p>
  </center>
</div>
`;
